src/server/rpc/procedures/geolocation/geolocation.js

```diff
@@ -58,7 +58,12 @@ if(!process.env.GOOGLE_GEOCODING_API) {
     };
 
 
-    // geocode an address and send back the details
+    /**
+     * Geolocates the address and sends back the coordinates
```

Comment:
I am not sure about the wording... We should probably come up with guidelines for these... "returns" might be better than "sends back"

Reply:
It isn't necessary to come up with guidelines for this PR but it is something that should be done